build_deploy.sh

```diff
@@ -14,6 +13,19 @@ COMMON_BUILDER=https://raw.githubusercontent.com/RedHatInsights/insights-fronten
 # ---------------------------
 
 set -ex
+
+
+# get correct config build per build
+ENV_BRANCH=$(sed "s/origin\///" <<< "$GIT_BRANCH")
+CHROME_CONFIG_BRANCH=""
+if [[ $ENV_BRANCH == "master" ]]
+then
+  CHROME_CONFIG_BRANCH="prod-beta"
+else
+  CHROME_CONFIG_BRANCH=$(sed "s/master/prod/" <<< "$ENV_BRANCH")
+fi
+export CHROME_CONFIG_BRANCH=$CHROME_CONFIG_BRANCH
```

Comment:
I don't think you have to write it like this, it should be enough to export the branch
```suggestion
if [[ $ENV_BRANCH == "master" ]]
then
  export CHROME_CONFIG_BRANCH="prod-beta"
else
  export CHROME_CONFIG_BRANCH=$(sed "s/master/prod/" <<< "$ENV_BRANCH")
fi
```